mithril-client/src/aggregator_client/certificate_client.rs

```diff
@@ -135,7 +135,9 @@ mod tests {
         http_client
             .expect_get_content()
             .return_once(move |_| {
-                Err(AggregatorHTTPClientError::RemoteServerLogical(String::new()))
+                Err(AggregatorHTTPClientError::RemoteServerLogical(anyhow!(
+                    String::new()
+                )))
```

Comment:
This should do it as well:
```suggestion
                Err(AggregatorHTTPClientError::RemoteServerLogical(anyhow!("an error".to_string())))
```